### PR TITLE
[@types/sequelize] Fix incorrect parameter in DefineIndexesOptions [4.28.0]

### PR DESCRIPTION
**Pull request by rohitsud**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<http://docs.sequelizejs.com/class/lib/model.js~Model.html>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by todd — APPROVED** [review]
🎉

**Comment by rohitsud** [comment]
> 🎉

Thanks @todd. Which release will this change land in? Do I have to create a PR for the new release too?

**Comment by todd** [comment]
@rohitsud Once a DT maintainer merges this (I don't have merge permissions), a new version of the Sequelize types should automatically be published within an hour or two. You don't need to do anything else for now except wait 😁

**Pull request merged**
